README.md

```diff
@@ -94,40 +93,6 @@ spec:
    rangeEnd: "FD:FF:FF:FF:FF:FF"
 ```
 
-## NMState
```

Comment:
Let's keep this section to point users to kubernetes-operator instead.

Reply:
Should we ? Maybe this has to be done at kubevirt docs more than CNAO @phoracek what doyou think ?

Reply:
I think it would be still worth mentioning nmstate for people who may be surprised it's gone.

(and we should add it to kubevirt user-guide too)